.codecheckignore

```diff
@@ -19,6 +19,7 @@
 ^hal/storage_abstraction
 ^hal/tests/TESTS/mbed_hal/trng/pithy
 ^hal/tests/TESTS/mbed_hal/trng/pithy
+^hal/tests/TESTS/pin_names
```

Comment:
I don't understand why...

Reply:
Astyle seems to be failing for `PinNames_test.h` which isn't actually used for anything other than pytest for `pinvalidate.py` so I think it's best we exclude it from astyle checks.

Reply:
or you correct file style....?

Reply:
ok yes it was just whitespace. I couldn't tell from the log in Travis so I thought astyle was failing for no reason. Fixed.